elasticsearch/start

```diff
@@ -149,7 +149,7 @@ mkdir -p /conf/scripts
 echo "Starting Elasticsearch with the options $OPTS"
 CMD="$ES_HOME/bin/elasticsearch $OPTS"
 if [ `id -u` = 0 ]; then
-  echo "Running as non-root..."
+  echo "Running as root..."
```

Comment:
I had to stop and think about this one, but I'm pretty sure the original comment is correct. Since the `DEFAULT_ES_USER` is "elasticsearch" [here](https://github.com/itzg/dockerfiles/blob/master/elasticsearch/Dockerfile#L19) then this block is switching from the container's root user to "elasticsearch".